### DISCUSSION TITLE
Sharing my spec for Radial and Circular Stacked Bar Charts

### DISCUSSION THREAD

**Discussion by guru-florida** [vega/vega#4001] [Show and tell]
No issue. Just thought I would share my spec for 2 radial like bar charts I made. :)  Thought you might want to include in the Examples.

**Background:** Data is based on WiFi interference (wifi or non-wifi) detected on 12 2.4GHz channels. I wanted a stacked chart of interference but a rectangular chart didnt really fit neatly into my UI (literally a square peg in a round hole). I started with a radial bar chart (RadialBarStacked.json) that had symbolic cues of the popular Wifi Logo. Looks great, but the perception of the inner bands arent equal with the outer bands so I instead moved to a spoked design that gave equal perception to each channel (SpokedBarStacked.json).

Vega json spec attached below.

Got the hang of this Vega spec. :)  So great!
![RadialBarStacked](https://user-images.githubusercontent.com/8075957/65521372-21440c80-deb7-11e9-9eca-05edcdcccfd5.png)
![SpokedBarStacked](https://user-images.githubusercontent.com/8075957/65521385-24d79380-deb7-11e9-98d7-4660bd870399.png)

[RadialAndSpokedBarStacked.zip](https://github.com/vega/vega/files/3647741/RadialAndSpokedBarStacked.zip)

**Comment by hydrosquall** [comment]
Thanks for sharing both the original design and the perceptually balanced alternative!

Would you be interested to have this graph linked in a community examples gallery?

**Comment by guru-florida** [comment]
Sure, sounds great!
